conn/physic/units.go

```diff
@@ -154,6 +154,44 @@ func (e ElectricResistance) String() string {
 	return nanoAsString(int64(e)) + "Ω"
 }
 
+// Set sets the ElectricResistance to the value represented by s. Units are to
+// be provided in Ohm/Ohms or Ω with an optional SI prefix (n,p,u,µ,m,k,M,G or
```

Comment:
I'd recommend to use quotes:
// be provided in "Ohm"/"Ohms" or "Ω" with an optional SI prefix: "n", "p", "u" or "µ", "m", "k", "M", "G" or
// "T".